### PR TITLE
[KIECLOUD-472] - Update RHPAM and RHDM templates and imagestreams on …

### PR DESCRIPTION
**Pull request by spolti**
…OpenShift Service Catalog for 7.10.0 release

Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

**Review by mtnbikenc — APPROVED** [review]
/lgtm

File: roles/openshift_examples/files/examples/x86_64/image-streams/dotnet_imagestreams.json [hunk 1]
```diff
@@ -14,134 +14,197 @@
...
+                            "sampleRef": "dotnet-5.0",
+                            "version": "5.0"
                         },
                         "referencePolicy": {
                             "type": "Local"
                         },
                         "from": {
                             "kind": "DockerImage",
-                            "name": "registry.redhat.io/dotnet/dotnet-22-rhel7:2.2"
+                            "name": "registry.access.redhat.com/ubi8/dotnet-50:5.0"
```
Comment (mtnbikenc) [inline roles/openshift_examples/files/examples/x86_64/image-streams/dotnet_imagestreams.json:60]: @spolti Can you comment as to why some of these image names were switch to the old host (registry.access.redhat.com)?  This has caused a bug and these image streams should probably be corrected for the new host.  https://bugzilla.redhat.com/show_bug.cgi?id=1964084

Comment (spolti) [reply roles/openshift_examples/files/examples/x86_64/image-streams/dotnet_imagestreams.json:60]: THanks for the heads up.

**Pull request merged**